tools/run-tests.py

```diff
@@ -197,11 +198,6 @@ def get_arguments():
 
 BINARY_CACHE = {}
 
-TERM_NORMAL = '\033[0m'
```

Comment:
Where are these defined?

Reply:
It's moved into common place so that to handling the difference between POSIX/Windows